activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+*   `ActiveSupport::CurrentAttributes.resets` now accepts a method name
+
+    The block API is still the recommended approach, but now both APIs are supported:
+
+    ```ruby
+    class Current < ActiveSupport::CurrentAttributes
+      resets { Time.zone = nil }
+      resets :reset_time_zone
```

Comment:
Tiny nitpick, I think the `after_reset` alias more clearly describes the callback:
```suggestion
      after_reset { Time.zone = nil }
      after_reset :reset_time_zone
```

Reply:
https://api.rubyonrails.org/classes/ActiveSupport/CurrentAttributes.html currently uses `resets`. So we'd need to change that too.

I think the current way makes sense personally - this is the action that happens *when* a reset happens; there's nothing else happening for it to go after.

Reply:
Yeah sorry, it's a tiny nitpick. I didn't know about `resets` (or `after_reset`) before this PR.
I just think "After reset, reset time zone" is more readable than "Reset reset time zone".
Similarly, it's unclear to me at a quick glance if `resets { Time.zone = "UTC" }`  sets the `Time.zone` to "UTC" or resets it a previous value or nil, or when it gets called.
There was some previous discussion as well:
https://github.com/rails/rails/pull/35063#issuecomment-458319091

But it doesn' really effect the PR (which I think is solid), so feel free to ignore it 😄